output.json

```diff
@@ -3999,6 +4037,308 @@
         }
       }
     },
+    "/workflow_runs": {
+      "post": {
+        "summary": "Create a workflow run.",
+        "operationId": "create_workflow_run",
+        "requestBody": {
+          "required": true,
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "required": [
+                  "applicant_id",
+                  "workflow_id"
+                ],
+                "properties": {
+                  "workflow_id": {
+                    "type": "string",
+                    "description": "The unique identifier for the Workflow."
+                  },
+                  "applicant_id": {
+                    "type": "string",
+                    "description": "The unique identifier for the Applicant."
+                  },
+                  "custom_data": {
+                    "type": "object",
+                    "description": "Optional object with custom input data to be used in the Workflow Run."
+                  },
+                  "link": {
+                    "type": "object",
+                    "description": "Optional object for the configuration of the Workflow Run link."
+                  }
+                }
+              }
+            }
+          }
+        },
+        "responses": {
+          "201": {
+            "description": "A Workflow Run object.",
+            "content": {
+              "application/json": {
+                "schema": {
```

Comment:
In the response of the POST we return the full workflow run object, so here you can reference the workflow_run resource.